test/test_runner/test_runner.cpp

```diff
@@ -114,25 +114,25 @@ bool TestRunner::checkLogicalPlan(std::unique_ptr<PreparedStatement>& preparedSt
     TestStatement* statement, size_t resultIdx, Connection& conn, uint32_t planIdx) {
     auto result = conn.executeAndAutoCommitIfNecessaryNoLock(preparedStatement.get(), planIdx);
     TestQueryResult& testAnswer = statement->result[resultIdx];
-    std::string expectedError;
+    std::string actualError;
```

Comment:
i don't understand why you have to rename expectederror to actual error?

Reply:
I talked to guodong about this. The names were actually switched up; expectederror *isn't* the expected error, it's *actually the actual error*.